inshpect.output

```diff
@@ -0,0 +1,2 @@
+# Includes
+
```

Comment:
This file doesn't belong in the repository.

Reply:
Okay, perfect!